packages/client/tsconfig.json

```diff
@@ -1,25 +1,45 @@
 {
```

Comment:
I wonder if server and client can use the same tsconfig... Probably not.

Reply:
Good question, unfortunately I don't know, for sure there are some properties that needs to differ, all the react stuff but we can keep the same level of strictness for sure! Let's see once we start for real